account_move_line_report_xls/report/account_move_line_xlsx.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="action_account_move_line_xlsx" model="ir.actions.report">
+        <field name="name">Export Selected Lines</field>
+        <field name="model">account.move.line</field>
+        <field name="type">ir.actions.report</field>
+        <field name="binding_type">action</field>
+        <field name="binding_model_id" ref="account.model_account_move_line" />
+        <field
+            name="report_name"
+        >account_move_line_report_xls.account_move_line_xlsx</field>
```

Comment:
```suggestion
        <field name="report_name">account_move_line_report_xls.account_move_line_xlsx</field>
```

Reply:
@Swapnesh-SerpentCS 
this strange syntax is due to the new OCA prettier xml tools, hence modifying it will not help since will be reformatted
@sbidoul 
stéphane, can you have a look  and correct the xml formatting ?

Reply:
there is not much I can do for that, except maybe allowing a longer line length for xml files

Reply:
+1 for longer line length on XML files